app/views/book_suggestions/new.html.erb

```diff
@@ -0,0 +1,47 @@
+<html>
+  <head>
+    <title>Suggestions</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+
+    <script type="text/javascript">
+      function new_suggestion(){
```

Comment:
Im not going to review frontend, but take care about the conventions! if in some case you are going to work with it, make sure that a frontend reviewer is going to review your code